src/test/api.js

```diff
@@ -0,0 +1,87 @@
+/*eslint no-invalid-this: "off" */
```

Comment:
You probably already know this, but if it's just about suppressing current errors and not enabling things, you can also just use:

/* eslint-disable no-invalid-this, any-other-settings, etc */